pkg/ingestor/parser/vuln/vuln.go

```diff
@@ -124,8 +130,11 @@ func parseMetadata(s *attestation_vuln.VulnerabilityStatement) *generated.ScanMe
 
 // TODO (pxp928): Remove creation of osv node and just create the vulnerability nodes specified
 func parseVulns(_ context.Context, s *attestation_vuln.VulnerabilityStatement) ([]*generated.VulnerabilityInputSpec,
-	[]assembler.VulnEqualIngest, error) {
+	[]assembler.VulnEqualIngest, []assembler.VulnMetadataIngest, error,
+) {
+	now := time.Now()
```

Comment:
Based on the discussion on the meeting, I do see that if we set the timestamp to be now, it will always get re-created in the graph DB as it does the field does not match. It would be better to capture a timestamp of the score from the scanner and use that so that we do not keep re-creating the `VulnMetadata` node multiple time. This is similar to the comment @lumjjb made. If we can keep the timestamp static per the OSV database, unless the score changes, the node will not get re-created.

Reply:
Once the change above is done, this is ready to merge! Thanks @hown3d!

Reply:
I've dropped the `time.Now()` here